src/main/java/cartographer/CartographerPlugin.java

```diff
@@ -67,7 +67,7 @@
 @PluginInfo(
     status = PluginStatus.RELEASED,
     packageName = MiscellaneousPluginPackage.NAME,
-    category = PluginCategoryNames.DECOMPILER,
+    category = PluginCategoryNames.COMMON,
```

Comment:
You may want to set this to PluginCategoryNames.DIAGNOSTIC which would align with the other two pull requests which address this and other issues

Reply:
Thanks for the feedback. Easy fix :)